pg_search/src/postgres/customscan/pdbscan/exec_methods/fast_fields/mixed.rs

```diff
@@ -269,40 +266,24 @@ impl ExecMethod for MixedFastFieldExecState {
                                     // Null datum.
                                     continue;
                                 }
-                                Err(_) => {
-                                    // TODO: panic? Is the fallthrough below actually necessary?
-                                }
+                                Err(_) => {}
```

Comment:
Are we swallowing an error in here? Shouldn't we surface it?

Reply:
Resolved in #2610.